docs/About/about_PSReadLine.md

```diff
@@ -0,0 +1,1410 @@
+---
+ms.date:  12/08/2018
+schema:  2.0.0
+locale:  en-us
+keywords:  powershell,cmdlet
+title:  About PSReadLine
+---
+# PSReadLine
```

Comment:
There are some big problems here:

* I can't tell what you've changed because this file isn't a renamed version of `about_PSReadLine.help.txt`.
* You've extracted the XML doc comments from the source? The XML doc comments will likely not be kept in sync with this file without automation.
* And speaking of automation - the build script for the module generates the bulk of this file from the XML. That automation also verifies all bindable functions are documented. Your change isn't detected by that automation because of the filename change.